src/index.html

```diff
@@ -1,14 +1,75 @@
 <!doctype html>
 <html lang="en">
+  <!-- #region HEAD -->
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <!-- #region Fonts -->
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <!-- #endregion -->
     <link rel="stylesheet" href="./style.css">
+
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <!-- #region NAV LINKS -->
+      <nav class="nav">
+        <ul class="nav-menu">
+          <li>
+            <a href="#" class="nav-links is-active">
+              Apple
+            </a>
+          </li>
+          <li>
+            <a href="#" class="nav-links">
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a href="#" class="nav-links">
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+            href="#"
+            class="nav-links"
+            data-qa="hover"
+            >
```

Comment:
```suggestion
            <a
              href="#"
              class="nav-links"
              data-qa="hover"
            >
```
please pay attention to your code formatting

Reply:
check everywhere